onecodex/lib/enums.py

```diff
@@ -89,3 +89,19 @@ class Linkage(BaseEnum):
 class OrdinationMethod(BaseEnum):
     Pcoa = "pcoa"
     Smacof = "smacof"
+
+
+class AnalysisType(BaseEnum):
+    Classification = "classification"
+    Functional = "functional"
+
+
+class FunctionalAnnotations(BaseEnum):
+    Pathways = "pathways"
+    MetaCyc = "metacyc"
+    EggNOG = "eggnog"
```

Comment:
style nit: for the "keys" in enums, we try to use camel-case that handles acronyms like this:

```
EggNog
Go
Ko
Ec
```

I know this looks weird but it'd match the naming styling of other things in the library and in mainline (e.g. OrdinationMethod.Pcoa, OrdinationMethod.Smacof)